syntax = "proto3";
package grpc;
option go_package = "./";

// StudentFaceRequest upload
message StudentFaceRequest {
  string detected_name = 1;
  float accuracy = 2;
  bytes image_data = 3;
}

// StudentFaceResponse
message StudentFaceResponse {
  bool success = 1;
  string message = 2;
}

// ImageUploadService
service ImageUploadService {
  rpc UploadImage (stream StudentFaceRequest) returns (StudentFaceResponse);
}